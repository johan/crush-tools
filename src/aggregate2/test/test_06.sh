test_number=06
description="auto-labels"

outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

$bin -k 1 -c 2 -s 3 -L "$test_dir/test.in" > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi

