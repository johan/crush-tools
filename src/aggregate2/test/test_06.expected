Text-1	Numeric-1-Sum	Text-2-Count
first text value	3	3
second text value	8	4
